autoload/phpactor/input.vim

```diff
@@ -60,15 +54,56 @@ function! phpactor#input#choice(label, choices, ResultHandler)
     call a:ResultHandler(choices[choice - 1])
 endfunction
 
+function! s:determineConfirmLabelFromPreference(choiceLabel, preference)
+    let buffer = []
```

Comment:
not sure about variable scoping here -- should we use a scope prefix for local vars?

Reply:
they are local by default apparently :relieved:

Reply:
Apparently (`:help l:` stands for edge cases but I cannot reproduce it, i.e. override `v:count` with local count), but it seems to be a good practice to explicitly define the scope (like method scope in PHP). If you use [vint](https://github.com/Vimjas/vint) (with [ALE](https://github.com/dense-analysis/ale) or [CoC](https://github.com/iamcco/diagnostic-languageserver/wiki/Linters) for example), you can notice that `ProhibitImplicitScopeBuiltinVariable` is disabled by default. This also does not look like common practice in most popular and advanced plugins.

Reply:
Nice, I didn't know there was a static analysis tool for VimL :)